configs/container.config

```diff
@@ -1,19 +1,19 @@
 process {
-    withLabel: hisat2       { container = 'nanozoo/hisat2:2.1.0--66dae66' } 
-    withLabel: minimap2     { container = 'nanozoo/minimap2:2.22--b178b67' } 
-    withLabel: python3      { container = 'nanozoo/python_rnaseq:3--23823db' } 
-    withLabel: deseq2       { container = 'nanozoo/deseq2:1.28.0--0df1612' } 
-    withLabel: fastp        { container = 'nanozoo/fastp:0.23.1--9f2e255' } 
-    withLabel: fastqc       { container = 'nanozoo/fastqc:0.11.8--fbfa1d7' } 
-    withLabel: subread      { container = 'nanozoo/subread:2.0.1--713a8e7' } 
-    withLabel: multiqc      { container = 'nanozoo/multiqc:1.11--9dfdee6' }
-    withLabel: nanoplot     { container = 'nanozoo/nanoplot:1.38.1--e303519' }
-    withLabel: sortmerna    { container = 'nanozoo/sortmerna:2.1b--ceea1a1' } 
-    withLabel: trinity      { container = 'trinityrnaseq/trinityrnaseq:2.13.2' } 
-    withLabel: stringtie    { container = 'nanozoo/stringtie:2.1.7--420b0db' } 
-    withLabel: busco        { container = 'ezlabgva/busco:v5.2.1_cv1' } 
-    withLabel: dammit       { container = 'nanozoo/dammit:1.2--b47259e' } 
-    withLabel: dammitDB     { container = 'nanozoo/dammit:1.2--b47259e' }
-    withLabel: basic_tools  { container = 'nanozoo/python_rnaseq:3.8--7a7808c' }
-    withLabel: rattle       { container = 'huxleys/rattle:1.0.0--24021329c8b365f21959c56ee1cfb693c768c14e' }
```

Comment:
hm @fischer-hub , will this remove rattle now from the master branch? I think we don't want this :) Sorry if the re-base to `master` causes confusion now